core-ui/src/components/Extensibility/ExtensibilityDetails.js

```diff
@@ -23,6 +30,11 @@ export const ExtensibilityDetailsCore = ({ resMetaData }) => {
 
   const detailsProps = usePrepareDetailsProps(urlPath, 'name');
 
+  const ajv = new Ajv();
+  if (!ajv.validate(extSchemas.details, resMetaData?.details)) {
+    throwConfigError(t('extensibility.errors'), { error: ajv.errors });
```

Comment:
1. `t` here comes from extensibility translations, not our busola ones so it won't work
2. there's no `extensibility.errors` in our translations, you can use `extensibility.error` (without s) instead, but that will only double the same error. Please use something more descriptive.